.github/workflows/dependencies-head.yml

```diff
@@ -57,14 +58,14 @@ jobs:
       with:
         python-version: ${{ matrix.python-version }}
 
-    # Use nightly SciPy wheels from Anaconda's PyPI
-    # c.f. https://twitter.com/ralfgommers/status/1419917265781334025
     - name: Install dependencies
       run: |
-        python -m pip install --upgrade pip setuptools wheel
-        python -m pip --no-cache-dir --quiet install --upgrade ".[all,test]"
-        python -m pip uninstall --yes scipy
-        python -m pip install --upgrade --index-url https://pypi.anaconda.org/scientific-python-nightly-wheels/simple scipy
+        python -m pip install uv
+        uv pip install --system --upgrade pip setuptools wheel
+        uv pip --no-cache --quiet install --system --upgrade ".[all,test]"
+        uv pip uninstall --system scipy
+        # uv wants to upgrade dependencies (numpy) to a dev release too, so don't --upgrade
+        uv pip install --system --index-url https://pypi.anaconda.org/scientific-python-nightly-wheels/simple scipy
```

Comment:
This was previously forcing an upgrade of `numpy` with `uv` as well, but just want to test `scipy` which should work with `numpy` `v1` or `v2`, and trying to test `numpy` `v2` will break `jax` at the moment.